docs: add example using object in Response.json to README (#77)
VeryGoodOpenSource/dart_frog: README.md

}
```

We can also return any Dart object in the `body` of the  `Response.json` constructor and it will be serialized correctly as long as it has a `toJson` method that returns a `Map<String, dynamic>`.

💡 _Tip: Check out [json_serializable](https://pub.dev/packages/json_serializable) to automate the `toJson` generation_.

```dart
import 'package:dart_frog/dart_frog.dart';

Response onRequest(RequestContext context) {
  return Response.json(
    body: User(name: 'Dash', age: 42),
  );
}

> 

Reviewer: ```suggestion
```
```

}
```

> We can also return any Dart object in the `body` of the  `Response.json` constructor as long as it has a `toJson` and `fromJson` function (for example, using [json_serializable](https://pub.dev/packages/json_serializable)):

Reviewer: ```suggestion
We can also return any Dart object in the `body` of the  `Response.json` constructor and it will be serialized correctly as long as it has a `toJson` method that returns a `Map<String, dynamic>`. Check out [json_serializable](https://pub.dev/packages/json_serializable) to automate the `toJson` generation.
```

> ```dart
> import 'package:dart_frog/dart_frog.dart';
> 
> Response onRequest(RequestContext context) {
>   return Response.json(
>     body: MyCustomResponsePayload('hello world!'),
>   );
> }
> 
> @JsonSerializable()
> class MyCustomResponsePayload {
>   const MyCustomResponsePayload(this.responseParam);
>   
>   factory MyCustomResponsePayload.fromJson(Map<String, dynamic> json) => _$MyCustomResponsePayloadFromJson(json);
>   
>   Map<String, dynamic> toJson() => _$MyCustomResponsePayloadToJson(this);
>   
>   final String responseParam;
> }
> ```

Reviewer: ```suggestion
```dart
import 'package:dart_frog/dart_frog.dart';

Response onRequest(RequestContext context) {
  return Response.json(
    body: User(name: 'Dash', age: 42),
  );
}
```

```dart
import 'package:json_annotation/json_annotation.dart';

part 'user.g.dart';

@JsonSerializable()
class User {
  const User({required this.name, this.age});
  
  final String name;
  final int age;
  
  Map<String, dynamic> toJson() => _$UserToJson(this);
}
```
```
